### DISCUSSION TITLE
The compiled OpenSSL program cannot interact with the CA server using CMP in an Alpine image, especially when accessing the CA server via a domain name.

### DISCUSSION THREAD

**Discussion by Hsince** [openssl/openssl#23817] [Q&A]
I found that when using the CMP function of OpenSSL 3.2.0 version, an error occurred: unable to resolve the domain name. When running OpenSSL 3.2.0 in the Alpine image, especially when using CMP, what are the minimum plugins that should be installed in the Alpine image? I do not need to compile the OpenSSL source code in Alpine.

**Comment by nhorman** [comment]
This sounds like more of a configuration error than an openssl bug.  Are you able to perform domain name resolution with other utilities?  I.e. can you use dig or another name resolution tool to resolve the host your cmp command is trying to resolve?

**Comment by tom-cosgrove-arm** [comment]
Should this issue be moved to a discussion?

**Comment by nhorman** [comment]
I think so, yes
